samples/javascript_typescript/12.nlp-with-luis/README.md

```diff
@@ -10,31 +10,30 @@ In this sample, we demonstrate how to call LUIS to extract the intents from a us
     ```bash
     git clone https://github.com/Microsoft/botbuilder-samples.git
     ```
-- In a console, navigate to samples/javascript_typescript/12.nlp-with-luis
+- In a console, navigate to `samples/javascript_typescript/12.nlp-with-luis`
     ```bash
     cd samples/javascript_typescript/12.nlp-with-luis
     ```
-- [Optional] Update the .env file under samples/javascript_typescript/12.nlp-with-luis with your botFileSecret
-    For Azure Bot Service bots, you can find the botFileSecret under application settings.
+- [Optional] Update the .env file under `samples/javascript_typescript/12.nlp-with-luis` with your `botFileSecret`. For Azure Bot Service bots, you can find the `botFileSecret` under application settings.
 - Install modules and start the bot
     ```bash
-    npm i & npm start
+    npm i && npm start
     ```
 
 # Testing the bot using Bot Framework Emulator
 [Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
 
-- Install the Bot Framework Emulator from [here](https://aka.ms/botframework-emulator)
+- Install the Bot Framework Emulator from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
```

Comment:
`- Install the [Bot Framework Emulator](https://github.com/Microsoft/BotFramework-Emulator/releases)`

Reply:
Done

Reply:
Done